docs/content/users/quickstart.md

```diff
@@ -494,10 +494,36 @@ The Laravel project type can be used for [StarterKits](https://laravel.com/docs/
 
 === "Composer"
 
+    Laravel defaults to SQLite, but we use MariaDB to better mimic a production environment:
+
     ```bash
     mkdir my-laravel-site && cd my-laravel-site
     ddev config --project-type=laravel --docroot=public
-    ddev composer create "laravel/laravel:^11"
+    ddev start
+    ddev composer create "laravel/laravel:^12"
+    ddev launch
+    ```
+
+=== "Composer (SQLite)"
+
+    To use the SQLite configuration provided by Laravel:
+
+    ```bash
+    mkdir my-laravel-site && cd my-laravel-site
+    ddev config --project-type=laravel --docroot=public --omit-containers=db --disable-settings-management=true
+    ddev start
+    ddev composer create "laravel/laravel:^12"
+    ddev launch
+    ```
+
+    To switch an existing Laravel project to SQLite:
+
+    ```bash
+    ddev config --project-type=laravel --docroot=public --omit-containers=db --disable-settings-management=true
+    ddev restart
+    ddev composer run-script post-root-package-install
+    ddev dotenv set .env --db-connection=sqlite
+    ddev composer run-script post-create-project-cmd
```

Comment:
`post-root-package-install` copies `.env.example` to `.env` if it doesn't exist, then we set the `sqlite` connection in `.env`, and finally run migrations in `post-create-project-cmd`.